nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/main/java/org/apache/nifi/web/security/oidc/OidcService.java

```diff
@@ -198,6 +198,7 @@ public void exchangeAuthorizationCode(final String oidcRequestIdentifier, final
         }
 
         final CacheKey oidcRequestIdentifierKey = new CacheKey(oidcRequestIdentifier);
+        // TODO: Retrieve the authorization code separately from the exchange
```

Comment:
It looks like the internal methods in the identity provider were refactored to make this separate, but `exchangeAuthorizationCode` is still the only public method, while the others are still private. Maybe translate the change to here as well.

Reply:
@alopresto I have separated the authorization from `exchangeAuthorizationCode`.